components/gong/actions/retrieve-transcripts-of-calls/retrieve-transcripts-of-calls.mjs

```
@@ -43,21 +50,102 @@ export default {
+          let currentTopic = null;
+          let formattedTranscript = "";
+
+          // Process each sentence maintaining chronological order
+          const allSentences = callTranscript.transcript.reduce((acc, segment) => {
+            const sentences = segment.sentences.map((sentence) => ({
+              ...sentence,
+              speakerId: segment.speakerId,
+              topic: segment.topic,
+            }));
+            return [
+              ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)